Documentation/project-docs/dogfooding.md

```diff
@@ -8,37 +8,37 @@ this experience. Make sure to consult this document often.
 
 ## Install prerequisites
 
-1. Acquire the latest nightly .NET Core SDK by downloading the zip or tarball listed in https://github.com/dotnet/cli/blob/master/README.md#installers-and-binaries (for example, https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-sdk-latest-win-x64.zip ) into a new folder.
+1. Acquire the latest nightly .NET Core SDK by downloading the zip or tarball listed in https://github.com/dotnet/core-sdk#installers-and-binaries (for example, https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-sdk-latest-win-x64.zip ) into a new folder for instance `D:\git\playground\dogfooding\dotnet`.
```

Comment:
Being a command line user, I use `c:\d` 😉

Reply:
Perhaps something shorter like `c:\dotnet` might be more readable as you're repeating it here

Reply:
done thank's!